src/Session/profile.py

```diff
@@ -71,8 +71,13 @@ def addFoods(self, foods):
             return 1
         return 0
 
-    def load(self, name):
-        pass
+    ''' Checks if self.name of food items are the same. '''
+    def __eq__(self, other):
+        return self.name == other.name and self._foodStorages == other._foodStorages 
+
+    ''' Checks if self.name of food items are not the same. '''
+    def __ne__(self, other):
+        return self.name == other.name or self._foodStorages != other._foodStorages
```

Comment:
I'm confused by what you're trying to do here? If it's to enable a way to compare profiles, I feel like their descriptions are off and the implementation of `__ne__` might be off.